### ISSUE TITLE
Wintergrasp quest givers still have wrong spawn point after recent fix

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5944]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After the recent fix for the wintergrasp npcs spawning where they die instead of where they are supposed to, the quest givers still spawn wherever they died, the lvl 76 guards tho are spawning where they are supposed to
(player Ilovekfc/Ilovebuttsex is deathgriping quest givers on walls then killing them there)
**Expected behaviour**: Tell us what should happen instead
Npcs shouldnt respawn where they died but where their spawn point is
**Steps to reproduce the problem**:

1. pull a quest giver around
2. kill it
3. see how it respawned where it was killed

**Include proofs for this behaviour**
![WoWScrnShot_010421_201623](https://user-images.githubusercontent.com/36212275/103579879-d69aa800-4ee1-11eb-852d-8c33a2aa23b9.jpg)
This SS was made today in the WG that happened 2 hrs after the restart when the fix was applied

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by RandomXD** [comment]
ok this seems to be fixed now after today's restart, quest givers are spawning at their usual location not where they died.

**Comment by Lothloryen** [comment]
Greetings,

Yes, I deployed a fix earlier with the maintenance. Thank you for the report.

**Comment by RandomXD** [comment]
there's still some issue with them cuz sometimes i see them spawned wrong, idk if they can still be pulled anywhere but for sure something isnt completly fixed
![WoWScrnShot_011421_150052](https://user-images.githubusercontent.com/36212275/104594322-aab5ba00-5679-11eb-89d0-edf88a2e0428.jpg)
![WoWScrnShot_011421_144648](https://user-images.githubusercontent.com/36212275/104594339-af7a6e00-5679-11eb-9273-e80485ccd9c9.jpg)
![WoWScrnShot_011021_151723](https://user-images.githubusercontent.com/36212275/104594350-b3a68b80-5679-11eb-8d09-7bfb5be3fc25.jpg)

**Comment by RandomXD** [comment]
![WoWScrnShot_062621_110819](https://user-images.githubusercontent.com/36212275/123506962-a0a7d680-d66f-11eb-9b4a-5f67d710d161.jpg)
26/6/2021 this fix seem to be gone since all npcs can be glitched like you see in the screenshot
